content/en/developers/integrations/create-an-integration-detection-rule.md

```diff
@@ -0,0 +1,146 @@
+
+{{< img src="developers/integrations/SIEM_detection_rule_bottom.png" alt="Steps 4 and 5 of a filled-out detection rule creation form" style="width:90%;" >}}
+
+For more information, see the documentation on [configuring a detection rule][7].
+
+## Understanding validation messages
+
+### Rule JSON parsing
+```
+File=<FILE_PATH> in collection=<COLLECTION> is an invalid JSON: error=<ERROR>
+```
+This error means that the JSON located at `<FILE_PATH>` is considered invalid JSON
+
+### Rule ID/Rule name
+```
+partnerRuleId is empty for rule name="<RULE_NAME>" - partnerRuleId=<NEW_RULE_ID> is available
+```
+A `partnerRuleId` is required for each rule and is missing. Use the generated `<NEW_RULE_ID>`. 
+
+```
+partnerRuleId=<RULE_ID> is in the incorrect format for rule name="<RULE_NAME>", it must follow the format=^[a-z0-9]{3}-[a-z0-9]{3}-[a-z0-9]{3}$ - partnerRuleId=<NEW_RULE_ID> is available
+```
+The rule name is not in the correct format. Use the generated `partnerRuleId: <NEW_RULE_ID>` to fix the issue.
+
+```
+Duplicate partnerRuleId=<RULE_ID> for rule name="<RULE_NAME>" - <RULE_ID_KEY> must be unique and it is already used in rule_ids="<RULE_IDS>" - <RULE_ID_KEY>=<NEW_RULE_ID> is available
+```
+Each `partnerRuleId` must be unique. The current ID is already being used. The newly generated `partnerRuleId` is available.
+
+```
+Duplicate name="<RULE_NAME>" for <RULE_ID_KEY>=<RULE_ID> - name must be unique.
+```
+Each rule name must be unique. The current name is already being used. Update the rule name to be unique.
+
+### MITRE tags
+```
+The rule with partnerRuleId=<RULE_ID> contains a MITRE tag tactic but it does not contain the tag `security:attack` is not present in, please add it
+```
+When a rule contains a MITRE tag `tactic:<TAG_VALUE>`, the tag `security:attack` must be added to the list of tags.
+
+```
+The MITRE tactic/technique tag=<TAG> for partnerRuleId=<RULE_ID> appears to be incorrect (i.e. it does not exist in the MITRE framework).
+```
+The listed tactic/technique tag `<TAG>` does not follow the [MITRE framework](https://attack.mitre.org/). Please select a different tag.
+
+### Cases
+```
+The case status <CASE_STATUS> for <RULE_ID_KEY>=<RULE_ID> is incorrect, it should be one of <STATUS_LIST>.
+```
+The case status must be either `CRITICAL`, `HIGH`, `MEDIUM`, `LOW`, or `INFO`.
+
+```
+The case ordering for partnerRuleId=<RULE_ID> is incorrect, please modify to order cases from the highest severity to the lowest.
+```
+Each rule definition must be ordered by decreasing severity. Please reorder the cases into `CRITICAL`, `HIGH`, `MEDIUM`, `LOW`, and `INFO`.
+
+### Source tags
+```
+source=<SOURCE> in the tags of the rule with partnerRule=<RULE_ID> is not supported by Datadog documentation.
+```
+Reach out to Datadog to address the issue.
+
+### Rule content validation/ rule update
+```
+<RULE_ID_KEY>=<RULE_ID> name="<RULE_NAME>" - error=<ERROR>
+```
+Reach out to Datadog to address the issue.
```

Comment:
For this one, the error message should be sufficient to address the issue.

Reply:
To match the existing pattern, I'd like to include what they can do to address it.